server/grpc_service.go

```diff
@@ -1685,3 +1686,60 @@ func checkStream(streamCtx context.Context, cancel context.CancelFunc, done chan
 	}
 	<-done
 }
+
+const globalConfigPath = "/global/config/"
+
+func (s *Server) StoreGlobalConfig(ctx context.Context, request *pdpb.StoreGlobalConfigRequest) (*pdpb.StoreGlobalConfigResponse, error) {
+	name := globalConfigPath + request.Name
+	value := request.Value
+	_, err := s.client.Put(ctx, name, value)
```

Comment:
Better wrap this part to `s.storage`, you can refer to the `UpdateGCSafePoint` implementation.

Reply:
but the path of storage always have an ID of cluster for prefix, so we couldn't warp it that way.

Reply:
No, `s.storage` operator `KV` will always have a `clusterID` prefix in the key.